tests/unit/test_oidc.py

```diff
@@ -0,0 +1,226 @@
+"""
+Unit tests for the oidc code in the frontend.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import ga4gh.frontend as frontend
+import tests.utils as utils
+from mock import patch
+import urlparse
+import oic
+import oic.oic.message as message
+
+RANDSTR = 'abc'
+OICCODE = "5GzAsW4Gfz8Km8PTuayFi6j2tA2VnPrHiWybL%2BkryEpr05D0OUKTGTi49Z54yl" \
+          "%2FIB5KsUzGwSg%2BFpx%2FpBt0aVc3QsvPlsJu6WGHIchJ55UU%3D"
+
+OICRESPONSE = {'scope': u'openid profile',
+               'state': RANDSTR,
+               'code': u'5GzAsW4Gfz8Km8PTuayFi6j2tA2VnPrHiWybL+'
```

Comment:
We have unicode literals set in the futures section, so the 'u' prefix shouldn't be necessary here (or anywhere else).